requirements/test.in

```diff
@@ -7,7 +7,6 @@ pytest-cov                # pytest extension for code coverage statistics
 pytest-django             # pytest extension for better Django support
 ddt                       # data-driven tests
 
-bok-choy                  # integration tests
 selenium                  # integration tests
```

Comment:
@farhan As you are removing selenium test cases can we also uninstall it from here?

Reply:
yes, i am removing, coming in next commit